specification/elasticsan/resource-manager/readme.md

```diff
@@ -0,0 +1,79 @@
+# storagepool
+
+> see https://aka.ms/autorest
+
+This is the AutoRest configuration file for storagepool.
+
+## Getting Started
+
+To build the SDKs for My API, simply install AutoRest via `npm` (`npm install -g autorest`) and then run:
+
+> `autorest readme.md`
+
+To see additional help and options, run:
+
+> `autorest --help`
+
+For other options on installation see [Installing AutoRest](https://aka.ms/autorest/install) on the AutoRest github page.
+
+---
+
+## Configuration
+
+### Basic Information
+
+These are the global settings for the storagepool.
+
+```yaml
+openapi-type: arm
+tag: package-2021-11-20-preview
+```
+
+### Tag: package-2021-08-22-preview
+
+These settings apply only when `--tag=package-2021-11-20-preview` is specified on the command line.
+
+```yaml $(tag) == 'package-2021-11-20-preview'
+input-file:
+  - Microsoft.ElasticSan/preview/2021-11-20-preview/elasticsan.json
+```
+
+---
+
+# Code Generation
+
+## Swagger to SDK
+
+This section describes what SDK should be generated by the automatic system.
+This is not used by Autorest itself.
+
+```yaml $(swagger-to-sdk)
+swagger-to-sdk:
+  - repo: azure-sdk-for-python-track2
+  - repo: azure-sdk-for-java
+  - repo: azure-sdk-for-go-track2
+  - repo: azure-sdk-for-js
+  - repo: azure-sdk-for-ruby
+    after_scripts:
+      - bundle install && rake arm:regen_all_profiles['azure_mgmt_storagepool']
+```
+
+## Go
+
+See configuration in [readme.go.md](./readme.go.md)
+
+## Python
+
+See configuration in [readme.python.md](./readme.python.md)
+
+## Ruby
```

Comment:
We do not support Ruby anymore. Please remove Ruby related config

Reply:
Removed Ruby SDK information.